src/advancedHMC_MCMC.jl

```diff
@@ -0,0 +1,459 @@
+"""
```

Comment:
this docstring needs to move to be right on top of the function, otherwise it won't work